### PR TITLE
k8s uaa service uses ClusterIP

### PR DESCRIPTION
**Pull request by Birdrock**
- Add simple ingress addon for minikube

[#171333164]

### CODE REVIEW

**Comment by Birdrock** [comment]
Up for discussion about approach. Ingress isn't part of our offering, but it is a developer convenience.

File: k8s/addons/ingress.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+apiVersion: extensions/v1beta1
```
Comment (Birdrock) [inline k8s/addons/ingress.yml:2]: Should add file separator.

File: k8s/addons/ingress.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+kind: Ingress
+metadata:
+  name: uaa-ingress
+  annotations:
+    nginx.ingress.kubernetes.io/ssl-redirect: \"false\"
```
Comment (Birdrock) [inline k8s/addons/ingress.yml]: Do we need this?

**Review by srwaggon — APPROVED** [review]

**Comment by joshuatcasey** [comment]
I like it. Have small, composable pieces.

**Comment by Birdrock** [comment]
Going to rebase/merge then. Thanks for taking a look.

**Pull request merged**